style.css

```diff
@@ -41,14 +41,79 @@ body {
   background-blend-mode: normal, overlay, normal;
 }
 
-#mobile-menu {
-  display: flex;
-  height: 78px;
-  align-items: center;
-  width: 20px;
-  background: url(images/ic_menu.png);
-  background-repeat: no-repeat;
-  margin: 10px;
+.humburger {
+  display: block;
+  cursor: pointer;
+  margin: 25px;
+  z-index: 11;
+}
+
+.mobile-menu-logo {
+  position: relative;
+  width: 70%;
+  top: 16px;
+}
+
+.mobile-menu-logo img {
+  width: 110%;
+}
+
+.bar {
+  display: block;
+  background-color: #000;
+  width: 30px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
```

Comment:
Great job in implementing the project requirement, according to your design, the hamburger drop-down doesn't seem to occupy the full height as it is seen in the design given making the background-image at the bottom to be cut-off partially from the screen. Kindly ensure that you implement this feature to be similar to what we have in the design, where the background-image can be clearly seen. You did really well with your design, it is clean and looks appealing but it will look better when you fix this issue 😄  

Figma design :
![image](https://github.com/ctokoli/personal-portfolio-site/assets/20537712/dfa99ea0-e691-47ba-bc9c-6c8098a5a7e9)

Your design :
![image](https://github.com/ctokoli/personal-portfolio-site/assets/20537712/1eb17b2d-4eee-41c4-a482-1bbed7748378)